getting-started/deploying-to-openshift.adoc

```diff
@@ -0,0 +1,235 @@
+= Deploying a BAMOE application to OpenShift using Quarkus
+include::../partials/attributes.adoc[]
+
+This guide provides instructions for deploying a business application to OpenShift using three different methods. Ensure that the following prerequisites are in place before proceeding.
+
+== Prerequisites
+
+* https://docs.openshift.com/container-platform/4.16/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli[OpenShift CLI (`oc`)]
+* https://docs.docker.com/manuals/[Docker]
+
+== Deployment Options
+
+=== 1. Build and Deploy on OpenShift
+
+Use this method to build and deploy the application directly on OpenShift.
+
+[.steps]
+
+. **Log in to OpenShift CLI:**
++
+[source,shell]
+----
+oc login
+----
+
+. **Add the OpenShift extension to the Quarkus project:**
++
+[source,shell]
+----
+mvn quarkus:add-extension -Dextensions='quarkus-openshift'
+----
+
+. **Update the `src/main/resources/application.properties` file to enable the creation of an OpenShift Route:**
++
+[source,properties]
+----
+# OpenShift
+quarkus.openshift.route.expose=true
+## Route TLS configuration:
+quarkus.openshift.route.tls.termination=edge
+quarkus.openshift.route.tls.insecure-edge-termination-policy=None
+----
```

Comment:
We'll have a list of all these in 9.1.1? It would be good to link to those for a more thorough explanation of what the properties do.

Reply:
Not the complete list, no... we're compiling all the configuration options for 9.2.0.

Reply:
We'll have to remember that when it is done then.

Reply:
Will we need to list all configurations? Including those from extensions from Quarkus (like the OpenShift one used here)?

Reply:
I believe @gitgabrio is working on that task